src/main/java/com/dumbdogdiner/stickyapi/common/util/MathUtil.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright (c) 2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util;
+
+import org.bukkit.Location;
+import org.bukkit.util.Vector;
+
+import java.text.CharacterIterator;
+import java.text.StringCharacterIterator;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * <p>
+ * Util class for commonly used math operations.
+ * </p>
+ */
+public class MathUtil {
+
+    private static final Random random = new Random();
+
+    /**
+     * Get a random number within a range
+     *
+     * @param max maximum value
+     * @return a random integer within the specified range
+     */
+    public static int rInt(int max) {
+        return random.nextInt(max);
+    }
+
+    /**
+     * Get a random number within a range
+     *
+     * @param min minimum value
+     * @param max maximum value
+     * @return a random integer within the specified range
+     * @throws IllegalArgumentException when min is greater than max
+     */
+    public static int rInt(int min, int max) {
```

Comment:
```suggestion
    public static int randonInt(int min, int max) {
```

This is also in NumberUtil.

Reply:
Marked getRandomNumber as deprecated since it's now in MathUtil (where it should be anyway)